packages/oasis-actions/src/helpers/flashloan/resolve-provider.ts

```diff
@@ -0,0 +1,30 @@
+import { providers } from 'ethers'
+
+import { Network, NetworkByChainId } from '../../../../../helpers/network'
+import { FlashloanProvider } from '../../types/common'
+
+export const resolveFlashloanProvider = async (provider: providers.Provider) => {
```

Comment:
I don't think this should be here at all. This code more or less is in the `network.ts`